StortSpelProjekt/Engine/src/Renderer/DX12Tasks/CopyOnDemandTask.cpp

```diff
@@ -31,6 +31,56 @@ void CopyOnDemandTask::SubmitTexture(Texture* texture)
 	m_Textures.push_back(texture);
 }
 
+void CopyOnDemandTask::UnSubmitMesh(Mesh* mesh)
+{
+	// Erase uploaded data
+	Resource* uploadResourceVertices = mesh->m_pUploadResourceVertices;
+	Resource* uploadResourceIndices = mesh->m_pDefaultResourceIndices;
+	auto it = m_UploadDefaultData.begin();
+
+	// Remove vertexThings
+	while (it != m_UploadDefaultData.end())
+	{
+		if (std::get<0>(*it) == uploadResourceVertices)
+		{
+			it = m_UploadDefaultData.erase(it);
+			break;
+		}
+		it++;
+	}
+
+	// Remove indexThings
+	it = m_UploadDefaultData.begin();
+	while (it != m_UploadDefaultData.end())
+	{
+		if (std::get<1>(*it) == uploadResourceIndices)
+		{
+			it = m_UploadDefaultData.erase(it);
+			break;
+		}
+		it++;
+	}
+}
+
+void CopyOnDemandTask::UnSubmitText(Text* text)
+{
+	// Erase uploaded data
+	Resource* uploadResource = text->m_pUploadResourceVertices;
```

Comment:
Same as above. Is this correct or should it be "Default"?

Reply:
see above